build-system/tasks/presubmit-checks.js

```diff
@@ -414,6 +414,13 @@ var forbiddenTermsSrcInclusive = {
   '\\.webkitConvertPointFromPageToNode(?!_)': bannedTermsHelpString,
   '\\.changeHeight(?!_)': bannedTermsHelpString,
   '\\.changeSize(?!_)': bannedTermsHelpString,
+  'insertAmpExtensionScript(?!\\s*\/\\*OK\\*\\/)': {
```

Comment:
Changed. So we don't want anyone to use this function, even when they set it to `insertAmpExtensionScript\*OK*\`?